gpkit/nomials.py

```diff
@@ -492,22 +490,14 @@ def __pow__(self, other):
 
 
 class Constraint(Posynomial):
-    """A constraint of the general form monomial > posynomial
-    Stored internally (exps, cs) as a single Posynomial (1 >= self)
+    """A constraint of the general form posynomial <= monomial
```

Comment:
Hmm, I do feel a bit strongly about it; all the sums in the flight / helicopter example will look strange, and even when you latex'd up your airfoil fit I think you put it as 1 >= ...